views/guide.lavender

```diff
@@ -392,7 +394,8 @@ block content
         pre
           code.language-jade.
             p
```

Comment:
@mowchan Yes to match the wiki: 

![pipe on wiki](https://cloud.githubusercontent.com/assets/11019935/6644094/d0004ce2-c987-11e4-9ffd-dd491d792f59.png)

![double pipe on page](https://cloud.githubusercontent.com/assets/11019935/6644098/d8d44d5a-c987-11e4-9937-1ec0d89acf48.png)